src/renderers/BaseRenderer.php

```diff
@@ -183,6 +183,11 @@
      */
     public $jsExtraSettings = [];
 
+    /**
+     * @var array List of the locations of registered JavaScript code block or files in right order.
+     */
+    public $jsPositions = [View::POS_HEAD, View::POS_BEGIN, View::POS_END, View::POS_READY, View::POS_LOAD];
```

Comment:
if we have to follow strict order of section this property must be private to prevent developers break something unintentionally

Reply:
> Property $jsPositions added to the case if someone has his own implementation of View

If someone has a `View` implementation, he can add `position` and he will have to redefine the renderers.
On the other hand, I agree with you and I can not imagine why this should be done - available positions more than enough.